City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal and Dependabot updates for 26.01.2022

This includes a Drupal core security update, module updates and updates to helfi_platform_config. It also includes updates based on dependabot warnings for helfi_paatokset and hds_web_components

**To test**
- Checkout branch
- Run `make fresh` or `drush cr;drush updb;drush cim -y`
- Check that all modules are up-to-date: https://helsinki-paatokset.docker.so/en/admin/reports/updates
- Check that all relevant npm commands (npm i, npm run dev and npm run build) work in the helfi_paatokset theme
- Check that all relevant npm commands (npm i;npm run build) work in teh hds_web_components module
- Click around the site and make sure nothing signifigant breaks
- Make sure the theme still works after the npm updates
- Make sure that hds web components still work after the npm updates
